cmd/adapter/k8s/k8s_test.go

```diff
@@ -27,125 +27,95 @@ import (
-		grFwdCfgs []*gravityForwarderCfg
-	}
-	tests := []struct {
-		name string
-		args args
-		want []*gravityForwarderCfg
-	}{
-		{name: "test1",
-			args: args{grFwdCfgs: []*gravityForwarderCfg{
-				{Metadata: struct {
-					Name string `yaml:"name"`
-				}{Name: "name1"}},
-				{Metadata: struct {
-					Name string `yaml:"name"`
-				}{Name: "name2"},
-					Spec: struct {
-						Address  string `yaml:"address"`
-						Protocol string `yaml:"protocol,omitempty"`
-					}{Address: "127.0.0.2", Protocol: ""},
-				}}},
-			want: []*gravityForwarderCfg{
-				{Metadata: struct {
-					Name string `yaml:"name"`
-				}{Name: "name2"},
-					Spec: struct {
-						Address  string `yaml:"address"`
-						Protocol string `yaml:"protocol,omitempty"`
-					}{Address: "127.0.0.2", Protocol: ""},
-				}}},
+	grFwdCfgs := []*gravityForwarderCfg{
+		{Metadata: struct {
+			Name string `yaml:"name"`
+		}{Name: "name1"}},
+		{Metadata: struct {
+			Name string `yaml:"name"`
+		}{Name: "name2"},
+			Spec: struct {
+				Address  string `yaml:"address"`
+				Protocol string `yaml:"protocol,omitempty"`
+			}{Address: "127.0.0.2", Protocol: ""},
+		}}
+
+	want := []*gravityForwarderCfg{
+		{Metadata: struct {
+			Name string `yaml:"name"`
+		}{Name: "name2"},
+			Spec: struct {
+				Address  string `yaml:"address"`
+				Protocol string `yaml:"protocol,omitempty"`
+			}{Address: "127.0.0.2", Protocol: ""},
+		}}
+
+	cli := &Client{
+		logger: log.WithField("test", "filterInvalidCfgs()"),
 	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			cli := &Client{
-				logger: log.WithField("test", "filterInvalidCfgs()"),
-			}
-			if got := cli.filterInvalidCfgs(tt.args.grFwdCfgs); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("Client.filterInvalidCfgs() = %v, want %v", got, tt.want)
-			}
-		})
+	if got := cli.filterInvalidCfgs(grFwdCfgs); !reflect.DeepEqual(got, want) {
+		t.Errorf("Client.filterInvalidCfgs() = %v, want %v", got, want)
 	}
 }
 
 func TestClient_mergeFwdConfigs(t *testing.T) {
-	type args struct {
-		lrFwdCfg  *forwarder.Config
-		grFwdCfgs []*gravityForwarderCfg
-		wCfgTmpl  *forwarder.WorkerConfig
+
+	wCfgTmpl := &forwarder.WorkerConfig{
+		Name: "default",
+		Pipe: &forwarder.PipeConfig{Name: "pipe"},
+		Sink: &sink.Config{Type: "syslog", Params: map[string]interface{}{
+			"Protocol": "tcp",
+		}},
 	}
-	tests := []struct {
-		name string
-		args args
-		want *forwarder.Config
-	}{
-		{name: "test1",
-			args: args{
-				//wCfgTmpl
-				wCfgTmpl: &forwarder.WorkerConfig{
-					Name: "default",
-					Pipe: &forwarder.PipeConfig{Name: "pipe"},
-					Sink: &sink.Config{Type: "syslog", Params: map[string]interface{}{
-						"Protocol": "tcp",
-					}},
+
+	grFwdCfgs := []*gravityForwarderCfg{
+		//0
+		{Metadata: struct {
+			Name string `yaml:"name"`
+		}{Name: "name1"},
+			Spec: struct {
+				Address  string `yaml:"address"`
+				Protocol string `yaml:"protocol,omitempty"`
+			}{Address: "127.0.0.1", Protocol: "udp"}},
+		//1
```

Comment:
nit: do you need this hint? If you used proper structs, it would look like this:
```go
gravityForwarderConfigs := []*gravityForwarderCfg{
{
	Metadata: Metadata{
		Name: "name1",
	},
	Spec: Spec{
		Address:  "127.0.0.1",
		Protocol: "udp",
	},
},
{
	Metadata: Metadata{
		Name: "name2",
	},
	Spec: Spec{
		Address:  "127.0.0.2",
		Protocol: "",
	},
}}
```
and it would not be so sore on the eye.

Reply:
applied different formatting, tried to make it more readable...